### PR TITLE
[DRAFT] Dragonscale Necklace Removal & Bandit Adjustment

### PR DESCRIPTION
**Pull request by AllCorruptingWyrm**
## About The Pull Request

After a request that has been long asked for, I deliver. Bandit at its core is a short lived antagonist role, They are supposed to be a Brotherhood but by making a bandit all about the gear and extremely overpowered gear (that stacks ontop of a statpack) you have the capacity to completely powergame as seen with bandits with a d-ring, dragonscale cloak and so on and so forth. 

## Why It's Good For The Game

To put it simply, bandit being powercrept is not healthy for the server because the townies need to be compensated, in turn causing an arms race. This is not okay and healthy for the server. Along with this, Bandit is supposed to be a thing you play with friends hence the whole Brotherhood aspect of it, they absolutely should not be super soldiers.

### CODE REVIEW

**Comment by AllCorruptingWyrm** [comment]
Also, they are losing the no examine text. This is why you wear hoods.

**Comment by Troopermoores** [comment]
I personally dont think bandits need a nerf i do think they need their examine text fixed but id be careful nerfing a role hardly played boss.

**Comment by AllCorruptingWyrm** [comment]
i'll look into the examine issue but 12+ stats with crossbows, plate, ontop of statpacks and stuff is... gross.

**Comment by Feralape** [comment]
This is definitely going to clash with my PR because I'm adding dungeon chests that generates randomized loots, one of them being the dragonscale necklace

**Comment by Elli-Skala** [comment]
I feel like removing hedge knight and putting it as " + bandit adjustment"  is pretty unclear

**Comment by AllCorruptingWyrm** [comment]
> I feel like removing hedge knight and putting it as " + bandit adjustment" is pretty unclear

Read the PR; This is made absurdly clear. Bandit is powercrept and hasn't been changed for our server/community.
